tools/verify_docker_gpu.sh

```
@@ -0,0 +1,371 @@
+  --service frame-compare-test
+
+Notes:
+  - Requires a Linux host with a working NVIDIA driver and NVIDIA Container Toolkit.
+  - Requires Docker Compose 2.30.0 or later for the gpu-nvidia Compose override.
+  - If Compose is older, the script prints a docker run --gpus all fallback command.
+EOF
+}
+
+readonly MIN_COMPOSE_GPUS_VERSION="2.30.0"
+readonly DEFAULT_SERVICE="frame-compare-test"
+readonly REPO_ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")/.." && pwd)"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)